02/styles/global.css

```diff
@@ -4,23 +4,46 @@
     box-sizing: border-box;
 }
 
-.a, .b, .c {
+.a,
+.b,
+.c {
     border: 3px solid transparent;
     min-height: 50px;
-
     display: flex;
     justify-content: center;
     align-items: center;
+    height: 50vh;
 }
 
```

Comment:
od razu dodałbym w tym miejscu ustawienia dla grid-a:
```
body {
    grid-template-areas: 
        "top"
        "middle"
        "bottom";
    display: grid;
}
```
Zwróć uwagę, że każdy wiersz to zawartość w cudzysłowie